.github/workflows/test_on_push.yml

```diff
@@ -155,24 +131,21 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
           python-version: 3.12
 
       - name: Set up uv
-        uses: yezz123/setup-uv@v4
+        uses: astral-sh/setup-uv@v5
         with:
-          uv-venv: ".venv"
+          version: "latest"
+          python-version: 3.12
 
-      - name: Install nox
-        uses: wntrblm/nox@2024.10.09
+      - name: Install dependencies
+        run: |
+          uv pip install copier jinja2_time
+          uv tool install nox
```

Comment:
```suggestion
```

This can also be replaced with `uvx nox -s docs`, in line 160.

Reply:
Fair enough, calling nox only once in this job.